src/en/docs/faq.md

```diff
@@ -24,5 +24,8 @@ Only administrators can add, edit or delete custom emoji. If you'd like to do ei
 ## "I want to develop a Bot."
 It is possible to develop a Bot using the Misskey API. Please, [see here](../docs/api).
 
-## Which service does the note translation function use?
+## "Which service does the note translation function use?"
 [DeepL](https://www.deepl.com/) is being used for this.
+
+## "How do you increase the base 100mb of storage for users?"
```

Comment:
This is very critical because I know a lot of people didn't sign up for my instance because of the 100mb default limit and there is 0 documentation explaining how to do this.